admin/c2cgeoportal_admin/templates/layertree.jinja2

```diff
@@ -0,0 +1,139 @@
+{% extends "layout.jinja2" %}
+
+{% block extra_links %}
+<link href="{{request.static_url('c2cgeoportal_admin:node_modules/jquery-treegrid/css/jquery.treegrid.css')}}" rel="stylesheet">
+{% endblock extra_links %}
+
+{% block extra_scripts %}
+<script type="text/javascript" src="{{request.static_url('c2cgeoportal_admin:node_modules/jquery-treegrid/js/jquery.treegrid.js')}}"></script>
+<script type="text/javascript" src="{{request.static_url('c2cgeoportal_admin:node_modules/jquery.cookie/jquery.cookie.js')}}"></script>
+{% endblock extra_scripts %}
+
+{% block content %}
+
+<div id="layertree">
+
+    <div id="layertree-actions" class="btn-group">
+        <button id="layertree-expand" type="button" class="btn btn-default">Expand all</button>
+        <button id="layertree-collapse" type="button" class="btn btn-default">Collapse all</button>
+    </div>
+
+    <table id="layertree-table" class="tree table table-hover">
+        {% macro render_node(node, parent_id=None) -%}
+        <tr class="{{ node.item_type }} treegrid-{{ node.id }}{{' treegrid-parent-{}'.format(parent_id) if parent_id else ''}}">
+            <td>
+                {% if node.item_type == 'theme' %}
```

Comment:
Is it possible to use a dictionnary for switch case ?

Reply:
I've forgot this, that is not needed anymore with layertree/delete route

Reply:
I've used a dict in the view controller for the edit routes. Concerning the icons, I do not want to do the same. Maybe we do something shorter using less.